### PR TITLE
Further Site Improvements on People Page

### PR DESCRIPTION
**Pull request by kmpaul**
This PR makes some additions and improvements to the Pythia Portal, mainly to the "People" page, but it also corrects an oversight in only listing the UCAR NSF award number on the main page.  I added more names to the list of people on the Pythia Team page and I added GitHub usernames to those people on the list.

Let me know if you think this looks good!

### CODE REVIEW

**Review by dopplershift — APPROVED** [review]

File: content/index.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Ecosystem and Cloud Computing to make sense of huge volumes of
 numerical scientific data.
 
 Project Pythia is a community effort led by NCAR and the University
-of Albany, and funded by NSF EarthCube award #2026899.
+of Albany, and funded by NSF EarthCube awards \#2026899 (NCAR) and
+\#2026863 (University of Albany).
```
Comment (brian-rose) [inline content/index.md:25]: University at Albany

File: content/pages/people.md [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
-- Kevin Paul, NCAR
-- Matthew C. Long, NCAR
-- Brian E. J. Rose, U. Albany
-- Kevin Tyle, U. Albany
+# Project Pythia Team
+
+## Project Administrators
+
+Project Pythia is adminstrated by the Principal Investigators of the
+NSF EarthCube awards \#2026899 (UCAR) and \#2026863 (University of Albany).
```
Comment (brian-rose) [inline content/pages/people.md:6]: University at Albany

File: content/pages/people.md [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+
+- John Clyne (NCAR) [[@clyne](https://github.com/clyne)]
+- Matthew C. Long (NCAR) [[@matt-long](https://github.com/matt-long)]
+- Ryan May (Unidata) [[@dopplershift](https://github.com/dopplershift)]
+- Kevin Paul (NCAR) [[@kmpaul](https://github.com/kmpaul)]
+- Brian E. J. Rose (University of Albany) [[@brian-rose](https://github.com/brian-rose])]
```
Comment (brian-rose) [inline content/pages/people.md:12]: University at Albany

File: content/pages/people.md [hunk 4]
```diff
@@ -1,8 +1,29 @@
...
+- Kevin Tyle (University of Albany) [[@ktyle](https://github.com/ktyle)]
```
Comment (brian-rose) [inline content/pages/people.md:13]: University at Albany

**Review by brian-rose — COMMENTED** [review]
Sorry to be pedantic, but I noted several places where the text "University of Albany" should be changed to "University at Albany".

I know it's a weird name, but that's just how it is!

**Review by clyne — APPROVED** [review]

**Comment by kmpaul** [comment]
Ah!  Thanks. I'll fix that.

**Comment by kmpaul** [comment]
Ok.  I believe that's now fixed.

**Review by andersy005 — APPROVED** [review]

File: content/pages/people.md [hunk 5]
```diff
@@ -1,8 +1,29 @@
...
+
+Project Pythia is adminstrated by the Principal Investigators of the
+NSF EarthCube awards \#2026899 (UCAR) and \#2026863 (University at Albany).
+
+- John Clyne (NCAR) [[@clyne](https://github.com/clyne)]
+- Matthew C. Long (NCAR) [[@matt-long](https://github.com/matt-long)]
+- Ryan May (Unidata) [[@dopplershift](https://github.com/dopplershift)]
+- Kevin Paul (NCAR) [[@kmpaul](https://github.com/kmpaul)]
+- Brian E. J. Rose (University at Albany) [[@brian-rose](https://github.com/brian-rose])]
+- Kevin Tyle (University at Albany) [[@ktyle](https://github.com/ktyle)]
```
Comment (brian-rose) [inline content/pages/people.md:13]: There's an extra bracket here which is breaking the link.

File: content/pages/people.md [hunk 6]
```diff
@@ -1,8 +1,29 @@
...
+## Project Administrators
+
+Project Pythia is adminstrated by the Principal Investigators of the
+NSF EarthCube awards \#2026899 (UCAR) and \#2026863 (University at Albany).
+
+- John Clyne (NCAR) [[@clyne](https://github.com/clyne)]
+- Matthew C. Long (NCAR) [[@matt-long](https://github.com/matt-long)]
+- Ryan May (Unidata) [[@dopplershift](https://github.com/dopplershift)]
+- Kevin Paul (NCAR) [[@kmpaul](https://github.com/kmpaul)]
+- Brian E. J. Rose (University at Albany) [[@brian-rose](https://github.com/brian-rose])]
```
Comment (brian-rose) [inline content/pages/people.md:12]: There's an extra bracket here which is breaking the link.

**Review by brian-rose — CHANGES_REQUESTED** [review]
All looks good except the broken link to my github

File: content/pages/people.md (hunk 6 above)
Comment (kmpaul) [reply content/pages/people.md]: Good catch!  I missed that.

**Comment by kmpaul** [comment]
Fixed.

**Review by brian-rose — APPROVED** [review]

File: content/pages/people.md [hunk 7]
```diff
@@ -1,8 +1,29 @@
...
+a great deal of ongoing work is being conducted and lead by:
+
+- Anderson Banihirwe (NCAR) [[@andersy005](https://github.com/andersy005)]
+- Drew Camron (Unidata) [[@dcamron](https://github.com/dcamron)]
+- Orhan Eroglu (NCAR) [[@erogluorhan](https://github.com/erogluorhan)]
+- Julia Kent (NCAR) [[@jukent](https://github.com/jukent)]
+
+## Project Contributors
+
+Anyone can contribute and participate in Project Pythia!  Please see our
```
Comment (erogluorhan) [inline content/pages/people.md:27]: "contribute to and ..." ?

**Review by erogluorhan — COMMENTED** [review]
This looks good overall, though, I made a few inline comments about typos.

File: content/pages/people.md [hunk 8]
```diff
@@ -1,8 +1,29 @@
...
+- Matthew C. Long (NCAR) [[@matt-long](https://github.com/matt-long)]
+- Ryan May (Unidata) [[@dopplershift](https://github.com/dopplershift)]
+- Kevin Paul (NCAR) [[@kmpaul](https://github.com/kmpaul)]
+- Brian E. J. Rose (University at Albany) [[@brian-rose](https://github.com/brian-rose)]
+- Kevin Tyle (University at Albany) [[@ktyle](https://github.com/ktyle)]
+
+## Project Members
+
+In addition to the administration provided by the Principal Investigators,
+a great deal of ongoing work is being conducted and lead by:
```
[the remark this answers is not in the document]
Comment (kmpaul) [reply content/pages/people.md:18]: Resolved in #14.  Thanks @erogluorhan!

File: content/pages/people.md (hunk 7 above)
Comment (kmpaul) [reply content/pages/people.md:27]: Resolved in #14.  Thanks @erogluorhan!

**Pull request merged**